pom.xml

```diff
@@ -45,9 +47,58 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+
+			<!-- HANA -->
+			<dependency>
+				<groupId>com.sap.cloud.db.jdbc</groupId>
+				<artifactId>ngdbc</artifactId>
+				<version>${ngdbc.version}</version>
+			</dependency>
 		</dependencies>
 	</dependencyManagement>
 
+	<profiles>
+		<!-- Local development, enables dynamic switching between SQLite and HANA -->
+		<profile>
+			<id>local</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+			<dependencies>
+				<dependency>
+					<groupId>org.xerial</groupId>
+					<artifactId>sqlite-jdbc</artifactId>
+				</dependency>
+
+				<dependency>
```

Comment:
Please remove these two dependencies from here, as they are not related to running the app locally and are anyways already included in the build.